### PR TITLE
Build Hex docs

### PR DESCRIPTION
**Pull request by wojtekmach**
TODO:

- [x] Create webhook
- [ ] Check hex logo on mobile/retina etc

### CODE REVIEW

File: priv/scripts/hex_docs.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  popd
+
+  if [ -f unversioned-docs ]; then
+    pushd unversioned-docs
+    aws s3 cp "${app}" "s3://hexdocs.pm/${app}" --recursive --cache-control "public,max-age=3600" --metadata "{\"surrogate-key\":\"docspage/${app}\",\"surrogate-control\":\"public,max-age=604800\"}"
+    fastly_purge $BOB_FASTLY_SERVICE_HEXDOCS "docspage/${app}"
+  fi
+}
+
+if [ "$1" == "push" ]; then
```
Comment (wojtekmach) [inline priv/scripts/hex_docs.sh]: I based this on https://github.com/hexpm/bob/blob/32fe561ceb73058c31afc8097f04770a8b69a372/priv/scripts/elixir_guides.sh#L33 but come to think of it, we should only build docs on `vMAJOR.MINOR.PATCH` tags, not on branches (notably not on master); what's a good place to make this check? here?

Comment (ericmj) [reply priv/scripts/hex_docs.sh]: I would do the check in the elixir job. We have a similar check when building OTP.

**Comment by ericmj** [comment]
> Check hex logo on mobile/retina etc

We should do this for the Elixir logo also.

**Comment by wojtekmach** [comment]
Hex logo looks decent, Elixir's is a bit blurry; will do some more tests tomorrow on bigger screen.

**Pull request merged**